libqtile/core/manager.py

```diff
@@ -108,6 +109,7 @@ def __init__(
 
         self.core.wmname = getattr(self.config, "wmname", "qtile")
         if config.main:
+            warnings.warn("Defining a main function is deprecated, use libqtile.qtile", DeprecationWarning)
```

Comment:
Should we mention `@hook.subscribe.startup_complete` also?

Reply:
(here and in the changelog)

Reply:
To remind people that `qtile` can also be modified after startup?

Reply:
If you try to play `libqtile.qtile` inside the config directly, IIRC most things won't be initiated yet, so you will very quickly end up with errors. But if you play with it inside `@hook.subscribe.startup_complete`, you have the guarantee that everything is instantiated.